extractor/src/main/java/org/schabi/newpipe/extractor/services/soundcloud/extractors/SoundcloudStreamExtractor.java

```diff
@@ -343,22 +390,21 @@ public String getLicence() {
         // Tags are separated by spaces, but they can be multiple words escaped by quotes "
         final String[] tagList = track.getString("tag_list").split(" ");
         final List<String> tags = new ArrayList<>();
-        String escapedTag = "";
+        final StringBuilder escapedTag = new StringBuilder();
```

Comment:
Is there a reason why ``StringBuilder`` is used here?

Reply:
Yes, there is one, a SonarLint warning:

<img src="https://user-images.githubusercontent.com/74829229/155210191-27d2206f-d198-45d5-b899-a0f88ae7d424.png" width="500px" alt="sonar_warning_string_concatenation_loops">